WordPress/Sniffs/Functions/FunctionRestrictionsSniff.php

```diff
@@ -13,8 +13,8 @@ class WordPress_Sniffs_Functions_FunctionRestrictionsSniff implements PHP_CodeSn
 	 * Exclude groups
 	 *
 	 * Example: 'switch_to_blog,user_meta'
-	 * 
-	 * @var string Comma-delimited group list 
+	 *
+	 * @var string Comma-delimited group list
```

Comment:
Please reset the changes to this file - you're not making any real changes here and the code style fixes are being handled in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/590